Packages/vcs/Lib/Canvas.py

```diff
@@ -5018,7 +5021,7 @@ def postscript(self, file, mode='r', orientation=None, width=None, height=None,
 
         # figures out width/height
         W, H = self._compute_width_height(
-            width, height, units)
+            width, height, units, ps=True)
```

Comment:
That was actually the reason for the flickering, it was going through that bit and changed the size